docs/content.zh/docs/dev/table/sql/queries/window-deduplication.md

```diff
@@ -0,0 +1,93 @@
+Otherwise, the optimizer won’t be able to translate the query.
+
+Flink uses `ROW_NUMBER()` to remove duplicates, just like the way of [Window Top-N query]({{< ref "docs/dev/table/sql/queries/window-topn" >}}). In theory, Window Deduplication is a special case of Window Top-N in which the N is one and order by the processing time or event time.
+
+The following shows the syntax of the Window Deduplication statement:
+
+```sql
+SELECT [column_list]
+FROM (
+   SELECT [column_list],
+     ROW_NUMBER() OVER (PARTITION BY window_start, window_end [, col_key1...]
+       ORDER BY time_attr [asc|desc]) AS rownum
+   FROM table_name) -- relation applied windowing TVF
+WHERE rownum = 1 [AND conditions]
+```
+
+**Parameter Specification:**
+
+- `ROW_NUMBER()`: Assigns an unique, sequential number to each row, starting with one.
+- `PARTITION BY window_start, window_end [, col_key1...]`: Specifies the partition columns which contain `window_start` and `window_end` columns i.e. the window deduplicate key.
+- `ORDER BY time_attr [asc|desc]`: Specifies the ordering column, it must be a [time attribute]({{< ref "docs/dev/table/concepts/time_attributes" >}}). Currently Flink supports [processing time attribute]({{< ref "docs/dev/table/concepts/time_attributes" >}}#processing-time) and [event time attribute]({{< ref "docs/dev/table/concepts/time_attributes" >}}#event-time). Ordering by ASC means keeping the first row, ordering by DESC means keeping the last row.
+- `WHERE rownum = 1`: The `rownum = 1` is required for Flink to recognize this query is deduplication.
+
+{{< hint info >}}
+Note: the above pattern must be followed exactly, otherwise the optimizer won’t be able to translate the query.
+{{< /hint >}}
+
+## Example
+
+The following example shows how to keep last record for every tumbling 10 minutes window.
+
+```sql
+Flink SQL> SELECT *
+  FROM (
+    SELECT bidtime, price, item, supplier_id, window_start, window_end, 
+      ROW_NUMBER() OVER (PARTITION BY window_start, window_end ORDER BY bidtime DESC) as rownum
+    FROM TABLE(
+               TUMBLE(TABLE Bid, DESCRIPTOR(bidtime), INTERVAL '10' MINUTES))
+  ) WHERE rownum <= 1;
++------------------+-------+------+-------------+------------------+------------------+--------+
+|          bidtime | price | item | supplier_id |     window_start |       window_end | rownum |
++------------------+-------+------+-------------+------------------+------------------+--------+
+| 2020-04-15 08:09 |  5.00 |    D |   supplier4 | 2020-04-15 08:00 | 2020-04-15 08:10 |      1 |
+| 2020-04-15 08:17 |  6.00 |    F |   supplier5 | 2020-04-15 08:10 | 2020-04-15 08:20 |      1 |
++------------------+-------+------+-------------+------------------+------------------+--------+
```

Comment:
it's better we can provide the source table ddl and source table data, which could help users understand it more easily